docs/ja/about.md

```diff
@@ -27,34 +28,37 @@ description: このプロジェクトに関する追加情報
 - [**Revolt**](https://github.com/revoltchat/api): オープンソースのユーザー優先チャットプラットフォーム
 - [**Spacebar**](https://github.com/spacebarchat): 無料でオープンソースの自ホスト可能な Discord 互換のチャット/音声/ビデオプラットフォーム
 - [**Supabase**](https://github.com/supabase/supabase): オープンソースの Firebase 代替
-- [**Twitter API**](https://github.com/twitterdev/twitter-api-typescript-sdk): Twitter API の公式 SDK
 
 ## プロジェクトの目標
 
+### openapi-typescript
+
 1. 任意の有効な OpenAPI スキーマを TypeScript 型に変換できるようにすること。どんなに複雑なスキーマでも対応可能です。
 2. 生成される型は静的に解析可能で、実行時の依存関係がない（ただし、[enums](https://www.typescriptlang.org/docs/handbook/enums.html) のような例外はあります）。
 3. 生成された型は、元のスキーマにできるだけ一致し、元の大文字形式などを保持します。
 4. 型の生成 は Node.js だけで実行可能であり、（Java、Python などは不要）どんな環境でも実行できます。
 5. ファイルからの OpenAPI スキーマのフェッチや、ローカルおよびリモートサーバーからのフェッチをサポートします。
 
-## 比較
-
-### vs. swagger-codegen
+### openapi-fetch
 
-openapi-typescript は、swagger-codegen の軽量で使いやすい代替手段として作成されており、Java ランタイムや OpenAPI サーバーを実行する必要はありません。また、大規模なクライアントサイドコードも生成しません。実際、openapi-typescript が生成するすべてのコードは、**実行時の依存関係がない静的型** であり、最大のパフォーマンスと最小のクライアント負荷を実現します。
+1. 型は厳密で、最小限のジェネリクスで OpenAPI スキーマから自動的に推論されるべきです。
+2. ネイティブの Fetch API を尊重しつつ、（`await res.json()` などの）ボイラープレートを削減すること。
+3. 可能な限り軽量で高性能であること。
 
-### vs. openapi-typescript-codegen
+### openapi-react-query
 
-openapi-typescript-codegen は、元の swagger-codegen の Node.js 代替手段ですが、実際には同じものです。openapi-typescript は、openapi-typescript-codegen と同様に、**実行時の依存関係がない** という利点を持っていますが、openapi-typescript-codegen は、スキーマの複雑さに応じて `250 kB` 以上になるかなり大きなバンドルを生成する可能性があります。
+1. 型は厳格であり、必要最小限のジェネリクスでOpenAPIスキーマから自動的に推論されるべきです。
+2. 元の `@tanstack/react-query` API を尊重しつつ、ボイラープレートを減らします。
+3. できるだけ軽量でパフォーマンスが高くなるようにします。
 
-### vs. tRPC
+## メインテナー
```

Comment:
my barely-JLPT5 proficiency led me to make this guess. I don’t feel confident trying to write entire sentences though (I would never run something through Google Translate)